controller/src/main/java/io/pravega/controller/store/index/HostIndex.java

```diff
@@ -1,17 +1,11 @@
 /**
- * Copyright (c) 2017 Dell Inc., or its subsidiaries.
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
```

Comment:
Why in some files are we still keeping the year and in others we are removing it?